src/DetailsView/components/settings-panel/settings/issue-filing/issue-filing-settings.tsx

```diff
@@ -15,7 +15,7 @@ export const IssueFilingSettings = NamedFC<SettingsProps>('IssueFilingSettings',
     );
 
     return (
-        <>
+        <div>
```

Comment:
is this change from `React.Fragment` (aka `<>`) to and empty `<div>` necessary?

Reply:
I do not think so; I'll revert this in the commit where I establish the constant